app/main/controllers/template/rtmedia-actions.php

```diff
@@ -166,19 +166,18 @@ function rtmedia_gallery_options() {
 		}
 
 		if ( ! empty( $output ) ) {
-			echo $output; // @codingStandardsIgnoreLine
+			echo $output;
```

Comment:
Don't we need to add `// phpcs:ignore <rule-name>`?

Reply:
Check at other places as well.

Reply:
I'll Address this feedback in new PR for the same files.